site/app/templates/grading/electronic/Details.twig

```diff
@@ -424,9 +424,12 @@
                 {% set btn_class = "btn-primary" %}
             {% elseif not graded_gradeable.isTaGradingComplete() and core.getUser().getGroup() < 4 or core.getUser().getGroup() == 4 and not graded_gradeable.isPeerGradingComplete() %}
                 {% set contents = "Grading Incomplete" %}
-            {% elseif graded_gradeable.isTaGradingComplete() or core.getUser().getGroup() == 4 and graded_gradeable.isPeerGradingComplete() %}
-                {% set contents = graded_gradeable.getTaGradedGradeable.getTotalScore() ~ " / " ~ graded_gradeable.getGradeable().getTaPoints() %}
+            {% elseif graded_gradeable.isTaGradingComplete() and core.getUser().getGroup() == 4 and graded_gradeable.isPeerGradingComplete() %}
+            {% set contents = graded_gradeable.getTaGradedGradeable.getTotalScore(core.getUser()) ~ " / " ~ graded_gradeable.getGradeable().getPeerPoints() %}
```

Comment:
This and line 431 are the same, so why the if statement? Can it be removed?

Reply:
The fix otherwise seems to work.

Reply:
it looked like when TA grading is in progress, the peer score was not being shown correctly.
Yes, it could be merged in a single else statement but i thought it made more sense to mention specific case because we missed it in the last PR.

Reply:
@jchulton Should I change anything with it?